documentation/run-strategies-template.ejs

```diff
@@ -4,16 +4,23 @@ layout: "default"
 isPage: true
 ---
 
+<? var top = docfiles.pop() ?>
+
 ## Run Manager Strategies
 
 The [Run Manager](../run-manager/) gives you control over run creation, depending on run state. You can select run creation stratgies (rules) for which runs end users of your project work with when they log in to your project.
 
-There are several strategies included in epicenter.js:
+There are several strategies included in Epicenter.js:
 
 <? docfiles.forEach(function (doc) { ?><? var moduleName = doc.filename.split('/').pop().split('.')[0].replace('-strategy', '') ?>
 * [<?= moduleName ?>](#<?= moduleName ?>)<? }); ?>
 
-You can also [create your own](#create-your-own).
+In addition, there are two strategies currently used only by the [Scenario Manager](../scenario-manager/):
```

Comment:
I wouldn't even mention this -- this is more of an implementation detail and of no practical use to someone using it externally